srcpkgs/tox/template

```diff
@@ -1,25 +1,26 @@
 # Template file for 'tox'
 pkgname=tox
-version=3.25.1
-revision=2
+version=3.28.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools_scm"
 depends="python3-filelock python3-packaging
  python3-pluggy python3-pytest python3-toml python3-virtualenv"
 checkdepends="${depends} python3-pip python3-pytest-mock
  python3-flaky python3-pathlib2 python3-freezegun"
 short_desc="Generic virtualenv management and test command line tool"
-maintainer="Piotr Wójcik <chocimier@tlen.pl>"
+maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://tox.readthedocs.io/en/latest/"
 changelog="https://tox.readthedocs.io/en/latest/changelog.html"
```

Comment:
```suggestion
homepage="https://tox.wiki/en/legacy/"
changelog="https://tox.wiki/en/legacy/changelog.html"
```
redirects, also switch branch for 3.x

Reply:
OK, even if ignoring 4.* wasn't intended to commit.